pkg/util/timeutil/time_zone_util.go

```diff
@@ -12,12 +12,17 @@ package timeutil
 
 import (
 	"fmt"
+	"regexp"
 	"strconv"
 	"strings"
 	"time"
 )
 
 const fixedOffsetPrefix string = "fixed offset:"
+const pattern = `(?i)(GMT|UTC)([+-])(\d{1,3}(:[0-5]\d){0,2})\b$`
```

Comment:
nit: making a minor tweak here to have more descriptive var names.
also changed regexp to support `UTC+3:5:6`.

Reply:
Wouldn't this regex accept an input like `utc+3:67:99`.  Ideally minutes and secs shouldn't be more than 59 right? This particular input won't be out of bounds

Reply:
`(?i)(GMT|UTC)([+-])(\d{1,3}(:[0-5]?\d){0,2})\b$`
This regex accepts cases like `UTC+3:5:6`

Reply:
ah that's true :) we want to allow single digits though, e.g. `GMT+3:5` works